_rules/SC3-1-2-lang-valid.md

```diff
@@ -27,7 +27,7 @@ The `lang` and `xml:lang` attributes have a [valid language subtag](#valid-langu
 
 ## Assumptions
 
-*There are currently no assumptions*
+The `lang` or `xml:lang` attribute is being used to comply to WCAG.
```

Comment:
I don't think this explains anything. Can you explain?

Reply:
@WilcoFiers fair point. It relates to this conversation and comment https://github.com/auto-wcag/auto-wcag/issues/348#issuecomment-456379859.

How about the following.

> This rule assumes that the presence of a `lang` or `xml:lang` attribute is being used to comply to WCAG. This rule doesn't test if the attribute is needed to comply to WCAG.